src/helpers/auth.js

```diff
@@ -0,0 +1,9 @@
+import { defaultClient } from "./client";
+
+export const login = async (credentials) =>
```

Comment:
I really like the fact of separating this file and the `client.js` file, but I'm not sure if the `helpers` directory name is well suited 🤔

Reply:
Ok. I'll make "api" directory for these file.